src/npc.cpp

```diff
@@ -1030,35 +1030,27 @@ const Skill* npc::best_skill() const
 void npc::starting_weapon(npc_class type)
 {
     const Skill* best = best_skill();
-    item sel_weapon;
-    if( best == nullptr ) {
-        // Fall through to random weapon
-    } else if (best->ident() == skill_bashing ) {
-        sel_weapon = random_item_from( type, "bashing" );
-    } else if (best->ident() == skill_cutting ) {
-        sel_weapon = random_item_from( type, "cutting" );
-    } else if (best->ident() == skill_stabbing ) {
-        sel_weapon = random_item_from( type, "stabbing" );
-    } else if (best->ident() == skill_throw ) {
-        sel_weapon = random_item_from( type, "throw" );
-    } else if (best->ident() == skill_archery ) {
-        sel_weapon = random_item_from( type, "archery" );
-    }else if (best->ident() == skill_pistol ) {
-        sel_weapon = random_item_from( type, "pistol", "guns_pistol_common" );
-    }else if (best->ident() == skill_shotgun ) {
-        sel_weapon = random_item_from( type, "shotgun", "guns_shotgun_common" );
-    }else if (best->ident() == skill_smg ) {
-        sel_weapon = random_item_from( type, "smg", "guns_smg_common" );
-    }else if (best->ident() == skill_rifle ) {
-        sel_weapon = random_item_from( type, "rifle", "guns_rifle_common" );
-    }else if (best->ident() == skill_launcher ) {
-        sel_weapon = random_item_from( type, "launcher" );
-    }
-
-    if (sel_weapon.is_null()){
-        sel_weapon = random_item_from( type, "weapon_random" );
-    }
-    weapon = sel_weapon;
+
+    // if NPC has no suitable skills default to bashing weapon
+    if( !best || best->ident() == skill_bashing ) {
```

Comment:
Bashing group includes (at high chances) semi-rare weapons like bats. Getting a bat would require having one nearby (suggests knowing how to swing it, ie. bashing skill) or looting it (requires getting out and avoiding conflict).

Cutting weapons include semi-rare, good weapons that would most likely not be possessed by someone not proficient in them.

Spears are easy to acquire and effective at low skill. Being able to survive without combat skills suggests some sort of survival sense, so crafting that spear shouldn't be a problem. Knives are also fine, because they're the obvious choice of anyone trying to kill human-shaped enemies, they work well at low levels and there's a reason to own them without being proficient in using them to attack (stabbing someone and cutting wood/meat/vegetation are totally different motions).

Also, it would be cool if there was a group for unarmed weapons, containing brass knuckles and the like (but not punch dagger or tiger claws, those are too "restricted").